cmds/boot/systemboot/main.go

```diff
@@ -98,6 +100,36 @@ func runIPMICommands() {
 
 }
 
+func sendBootSEL(sequence string) {
+	var bootErr ipmi.Event
+	var OEM_NTS_TYPE uint8 = ipmi.OEM_NTS_TYPE
+
+	ipmi, err := ipmi.Open(0)
+	if err != nil {
+		log.Printf("Failed to open ipmi device to send SEL %v", err)
+		return
+	}
+	defer ipmi.Close()
+
+	switch sequence {
+	case "fbnetboot":
+		bootErr.RecordID = 0x0000
+		bootErr.RecordType = OEM_NTS_TYPE
+		bootErr.OEMNontsDefinedData = [13]uint8{
+			0x28,
+			0, 0, 0, 0,
+			0x00,
+			0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,
+		}
+		if err := ipmi.LogSystemEvent(&bootErr); err != nil {
+			log.Printf("SEL recorded: %s fail\n", sequence)
+		}
+	case "localboot":
```

Comment:
If this is not being used right now I would remove it.

Reply:
Removed since cases are not used.